pkg/certs/cert-inspection/certgraphanalysis/certkeypair.go

```diff
@@ -108,6 +108,8 @@ func toCertKeyMetadata(certificate *x509.Certificate) certgraphapi.CertKeyMetada
 		},
 		SignatureAlgorithm: certificate.SignatureAlgorithm.String(),
 		PublicKeyAlgorithm: certificate.PublicKeyAlgorithm.String(),
+		NotBefore:          certificate.NotBefore.String(),
```

Comment:
aren't we using `time.RFC3339` for formatting ?

Reply:
Good point, updated